src/conductor/client/automator/task_runner.py

```diff
@@ -85,6 +86,16 @@ def __poll_task(self) -> Task:
                 self.metrics_collector.record_task_poll_time(
                     task_definition_name, time_spent
                 )
+        except ApiException as e:
+            if self.metrics_collector is not None:
+                self.metrics_collector.increment_task_poll_error(
+                    task_definition_name, type(e)
+                )
+            logger.error(f'{task_definition_name} worker cannot connect to conductor API, status code: {e.status}, detail: {e.body}')
```

Comment:
This will be too many errors since polling happens quite frequently.

Reply:
changed it to info, is that also too much?